docs/advanced-topics/serialization/serialization-intro.md

```diff
@@ -0,0 +1,7 @@
+---
+id: serialization-intro
+title: Introduction 
```

Comment:
changing, can't have dupe titles

Reply:
Again perhaps i should have it as introduction to serialization

Reply:
Left as About